code/web/src/modules/StyleSurvey/StyleSurvey.js

```diff
@@ -189,33 +216,40 @@ class StyleSurvey extends Component {
 						maxWidth: '300px',
 					}}>
 					<ImageTile
-						onClick={e => {
-							this.collectAnswer(e);
-						}}
 						key={i}
 						data-category={image.style}
-						id={i}
 						width={'100%'}
             height={'250px'}
 						image={image.src}
-						style={{ boxShadow: level1, backgroundSize: 'cover' }}
+						style={{ boxShadow: image.selected ? glowShadow : level1, backgroundSize: 'cover' }}
```

Comment:
nice catch with adding a new shadow property in the `ui/typography`  file, even if the document is a little complex in the way the data is manipulated and pass to different components, it makes things more readable this way.